src/lib/select/option/option-foundation.ts

```diff
@@ -24,12 +24,13 @@ export class OptionFoundation implements IOptionFoundation {
   constructor(private _adapter: IOptionAdapter) {}
 
   /** Gets/sets the value of this option. */
-  public get value(): string {
+  public get value(): any {
     return this._value;
   }
-  public set value(value: string) {
+  public set value(value: any) {
     if (this._value !== value) {
       this._value = value;
+      this._adapter.emitHostEvent(OPTION_CONSTANTS.events.VALUE_CHANGE, this._value);
```

Comment:
Just calling this out because I don't necessarily love dispatching an event from a setter, but aside from monkey patching the `value` property on all `<forge-option>` elements, this seemed like the best approach. Especially given that this is an internal use only event that is used for notification purposes. Open to alternative suggestions.